node/resolvers/items.ts

```diff
@@ -194,10 +197,13 @@ export const mutations = {
       })
     }
 
-    const newOrderForm = await clients.checkout.updateItems(
+    const newOrderForm = await clients.checkout.addItem(
```

Comment:
`updateItems` is being used somewhere else? If not, you can delete this function from the client, so we won't keep unused code

Reply:
I thought the idea was to use the `addItems` while the route in`updateItems` is not using the contextual pipeline and change it back when it's implemented. Should I still delete it?

Reply:
Hmmm, I see. If you can confirm this, just add a comment talking about this on top of the `updateItems`

Reply:
Just in case we take some time to do this or someone delete it by mistake in a future PR

Reply:
We don't plan on going back so it will be deleted 😄